### ISSUE TITLE
Unable to build package Manjaro 18.0.4

### ISSUE THREAD

**Issue by ColonialDagger** [Francesco149/sharenix#34]
Both cloning this package with git and installing the package from AUR result in the same error when installing and aborts the process:

```
[user@desktop sharenix-git]$ makepkg -si
==> Making package: sharenix-git r146.3ce23df-2 (Sun 21 Apr 2019 05:55:58 AM EDT)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Cloning sharenix git repo...
Cloning into bare repository '/home/bitwise/sharenix-git/sharenix'...
remote: Enumerating objects: 701, done.
remote: Total 701 (delta 0), reused 0 (delta 0), pack-reused 701
Receiving objects: 100% (701/701), 170.39 KiB | 1.69 MiB/s, done.
Resolving deltas: 100% (476/476), done.
==> Validating source files with md5sums...
    sharenix ... Skipped
==> Extracting sources...
  -> Creating working copy of sharenix git repo...
Cloning into 'sharenix'...
done.
==> Starting prepare()...
==> Starting pkgver()...
==> Updated version: sharenix-git r160.5a0d92d-1
==> Starting build()...
# github.com/mattn/go-gtk/pango
/usr/bin/ld: $WORK/b096/_x002.o: undefined reference to symbol 'g_object_unref'
/usr/bin/ld: /usr/lib/libgobject-2.0.so.0: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
==> ERROR: A failure occurred in build().
    Aborting...
```

```
[user@desktop ~]$ yay sharenix
1 aur/sharenix-git r146.3ce23df-1 (+1 0.43%) 
    file uploading tool with ShareX-compatible config
==> Packages to install (eg: 1 2 3, 1-3 or ^4)
==> 1
:: Checking for conflicts...
:: Checking for inner conflicts...
[Aur: 1]  sharenix-git-r146.3ce23df-1

  1 sharenix-git                             (Build Files Exist)
==> Packages to cleanBuild?
==> [N]one [A]ll [Ab]ort [I]nstalled [No]tInstalled or (1 2 3, 1-3, ^4)
==> 
:: PKGBUILD up to date, Skipping (1/1): sharenix-git
  1 sharenix-git                             (Build Files Exist)
==> Diffs to show?
==> [N]one [A]ll [Ab]ort [I]nstalled [No]tInstalled or (1 2 3, 1-3, ^4)
==> 
:: Parsing SRCINFO (1/1): sharenix-git
==> Making package: sharenix-git r146.3ce23df-2 (Sun 21 Apr 2019 05:59:28 AM EDT)
==> Retrieving sources...
  -> Updating sharenix git repo...
Fetching origin
==> Validating source files with md5sums...
    sharenix ... Skipped
==> Cleaning up...
==> Making package: sharenix-git r146.3ce23df-2 (Sun 21 Apr 2019 05:59:28 AM EDT)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Updating sharenix git repo...
Fetching origin
==> Validating source files with md5sums...
    sharenix ... Skipped
==> Removing existing $srcdir/ directory...
==> Extracting sources...
  -> Creating working copy of sharenix git repo...
Cloning into 'sharenix'...
done.
==> Starting prepare()...
==> Starting pkgver()...
==> Updated version: sharenix-git r160.5a0d92d-1
==> Sources are ready.
==> Making package: sharenix-git r160.5a0d92d-1 (Sun 21 Apr 2019 05:59:29 AM EDT)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Starting pkgver()...
==> Starting build()...
# github.com/mattn/go-gtk/pango
/usr/bin/ld: $WORK/b096/_x002.o: undefined reference to symbol 'g_object_unref'
/usr/bin/ld: /usr/lib/libgobject-2.0.so.0: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
==> ERROR: A failure occurred in build().
    Aborting...
Error making: sharenix-git

```

**Comment by Francesco149** [comment]
i think some dependencies are missing from the pkgbuild

try installing cairo, glib2, and pango

**Comment by ColonialDagger** [comment]
I installed the above packages. However, the same error occurs.

**Comment by Francesco149** [comment]
this might be an issue with go-gtk. will try to help them fix it https://github.com/mattn/go-gtk/issues/395

**Comment by Francesco149** [comment]
it looks like arch fixed this in pango 1.43.0-2 but manjaro stable hasnt picked up the change yet. https://git.archlinux.org/svntogit/packages.git/commit/trunk?h=packages/pango&id=d0516bfe5e6fda19d1ef5560509001bfdf6e846b

you can wait until manjaro stable gets this version of pango (probably a few days) or [switch to unstable](https://wiki.manjaro.org/index.php/Access_the_Unstable_Repositories)

**Comment by Sphiinx** [comment]
@Francesco149 @ColonialDagger Looks like this issue is resolved. I'm able to build the package from the AUR.

Can you guys confirm?

**Comment by Francesco149** [comment]
@Sphiinx yeah it should be fixed since pango 1.43.0-2 on both manjaro and arch
